keyboards/spacetime/keymaps/kyleterry/keymap.c

```diff
@@ -0,0 +1,168 @@
+    _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,     _______, _______, KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+    _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,    _______, _______, _______, _______, _______, KC_VOLD, KC_VOLU,
+        _______,                        _______, _______, _______,   _______, _______, _______,                        _______
+  ),
+
+  [_ADJUST] = LAYOUT(
+    _______, _______, KC_PGUP, _______, _______, _______, SQWER,     _______, _______, _______, KC_INS,  _______, KC_PSCR, _______,
+    _______, KC_HOME, KC_PGDN, KC_END,  _______, _______, SGAME,     _______, KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______,   _______, _______, _______, _______, _______, _______, _______,
+        _______,                        _______, _______, _______,   _______, _______, _______,                        _______
+  ),
+
+  [_NUM] = LAYOUT(
+    _______, _______, _______, _______, _______, _______, _______,   _______, _______, _______, KC_7,    KC_8,    KC_9,    KC_ASTR,
+    _______, _______, _______, _______, _______, _______, _______,   _______, _______, _______, KC_4,    KC_5,    KC_6,    KC_MINS,
+    _______, _______, _______, _______, _______, _______, _______,   _______, _______, _______, KC_1,    KC_2,    KC_3,    KC_PLUS,
+        _______,                        _______, _______, _______,   _______, _______, _______,                        KC_0
+  )
+};
+
+uint32_t layer_state_set_user(uint32_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  return true;
+}
+
+void matrix_init_user(void) {
+
+}
+
+void matrix_scan_user(void) {
+
+}
+
+void led_set_user(uint8_t usb_led) {
+
+}
+
+void tap_key(uint16_t keycode) {
+  register_code  (keycode);
+  unregister_code(keycode);
+}
+
+void shift_key(uint16_t keycode) {
+  register_code  (KC_LSFT);
+  tap_key        (keycode);
+  unregister_code(KC_LSFT);
+}
+
+void control_key(uint16_t keycode) {
+  register_code  (KC_LCTL);
+  tap_key        (keycode);
+  unregister_code(KC_LCTL);
+}
+
+void td_common(qk_tap_dance_state_t *state, void *user_data) {
+  switch (state->count) {
+    case 1:
+      /* this case handles ctrl+o which is my tmux prefix
+       */
+      control_key(KC_O);
+      reset_tap_dance(state);
+      break;
+    case 2:
+      /* this case handles shift+insert which is a common way
+       * for me to paste text in linux
+       */
+      shift_key(KC_INS);
+      reset_tap_dance(state);
+      break;
+  }
+}
+
+void td_media(qk_tap_dance_state_t *state, void *user_data) {
+  switch (state->count) {
+    case 1:
+      tap_key(KC_MPLY);
+      reset_tap_dance(state);
+      break;
+    case 2:
+      tap_key(KC_MUTE);
+      reset_tap_dance(state);
```

Comment:
Certain keys, like the media keys may actually need a small delay between taps. 

If you `#define TAP_CODE_DELAY 10` to your config.h file, it should help with any "oddities" you may experience with the media keys.